code/controllers/subsystems/economy.dm

```diff
@@ -58,9 +58,9 @@ var/datum/controller/subsystem/economy/SSeconomy
 	T.target_name = station_account.owner_name
 	T.purpose = "Account creation"
 	T.amount = 75000
-	T.date = "2nd April, 2454"
+	T.date = "13th May, 2461"
```

Comment:
If this date has to reflect IRL day/month. You can grab server date var for this.

Reply:
It's not, just the day of the change.